procedural/scheduling/src/main/kotlin/gov/nasa/ammos/aerie/procedural/scheduling/simulation/SimulateOptions.kt

```diff
@@ -0,0 +1,9 @@
+package gov.nasa.ammos.aerie.procedural.scheduling.simulation
+
+/** Configuration for simulation. */
+/* data */ class SimulateOptions(
+    // The following two options will be uncommented when checkpoint simulation is released.
+//    val checkPointGeneration: CheckpointGeneration = CheckpointGeneration.None,
+//    val checkpointRetention: CheckpointRetention = CheckpointRetention.All,
+    val pause: PauseBehavior = PauseBehavior.AtEnd,
```

Comment:
Will come back to this in V2 to determine if the user is able to explicitly use checkpoint simulation